grisera_api/scenario/scenario_service.py

```diff
@@ -25,114 +25,131 @@ def save_scenario(self, scenario: ScenarioIn):
         Returns:
             Result of request as scenario object
         """
-        activities = [self.activity_service.save_activity(activity=activity) for activity in scenario.activities]
-        self.graph_api_service.create_relationships(scenario.experiment_id, activities[0].id, 'hasActivity')
-        [self.graph_api_service.create_relationships(activities[index-1].id, activities[index].id, 'next')
-         for index in range(1, len(activities))]
-
-        return ScenarioOut(experiment_id=scenario.experiment_id, activities=activities)
-
-    def add_activity(self, previous_id: int, activity: ActivityIn):
+        activity_executions = [
+            self.activity_execution_service.save_activity_execution(activity_execution=activity_execution) for
+            activity_execution in scenario.activity_executions]
+        self.graph_api_service.create_relationships(scenario.experiment_id, activity_executions[0].id,
+                                                    'hasActivityExecution')
+        [self.graph_api_service.create_relationships(activity_executions[index - 1].id, activity_executions[index].id,
+                                                     'nextActivityExecution')
```

Comment:
I have just noticed that there is also a relationship called isNextOfActivityExecution which is the inverse of nextActivityExecution